core/src/main/java/com/netflix/conductor/service/WorkflowBulkServiceImpl.java

```diff
@@ -164,4 +164,24 @@ public BulkResponse terminate(List<String> workflowIds, String reason) {
         }
         return bulkResponse;
     }
+
+    @DeleteMapping("/workflows/{workflowIds}/remove")
+    public BulkResponse delete(List<String> workflowIds) {
+        BulkResponse bulkResponse = new BulkResponse();
+        ExecutionService executionService = new ExecutionService();
+        for(String workflowId : workflowIds) {
+            try {
+                executionService.remove(workflowId, true);
+                bulkResponse.appendSuccessResponse(workflowId);
+            } catch (Exception e) {
+                LOGGER.error(
+                        "bulk delete exception, workflowId {}, message: {} ",
+                        workflowId,
+                        e.getMessage(),
+                        e);
+                bulkResponse.appendFailedResponse(workflowId, e.getMessage());
+            }
+        }
+        return bulkResponse;
+    }
```

Comment:
Should be in com/netflix/conductor/rest/controllers/WorkflowBulkResource.java
Pull the repo with my changes and call workflowBulkService.deleteWorkflow